### DISCUSSION TITLE
Show the "Your Proposal"  in the "Filters"

### DISCUSSION THREAD

**Discussion by Angus0430** [snapshot-labs/snapshot#1087] [Ideas]
I can not find the proposal created by myself in the UI now.  It is very inconvenient to check the voting results，when there are more proposals ,or in different spaces.

Snapshot could be add this feature，show the "Your Proposal"  in the "Filters" .

![1637388624(1)](https://user-images.githubusercontent.com/90793534/142716523-05cd14fb-530e-43a4-a37d-b9d4cbd02f2c.png)

**Comment by metrox-eth** [comment]
Nice idea, thanks !
